package.json

```diff
@@ -35,7 +35,7 @@
     "pa11y": "^5.2.0",
     "pa11y-ci": "^2.3.0",
     "path": "^0.12.7",
-    "postcss-cli": "^4.1.0",
```

Comment:
This was causing the build to fail, so I upgraded it and did a quick check. It looks to be in order but if you could double check, that would be great.

Reply:
This wasn't the cause of the failure but I would rather keep the upgrade now.